lib/html_cleaner.rb

```diff
@@ -158,10 +158,14 @@ def sanitize_value(field, value)
       if ArchiveConfig.FIELDS_ALLOWING_CSS.include?(field.to_s)
         transformers << Sanitize::Transformers::ALLOW_USER_CLASSES
       end
-      # the screencast field shouldn't be wrapped in <p> tags
-      unless field.to_s == "screencast"
+      # Now that we know what transformers we need, let's santize the unfrozen value
+      if ArchiveConfig.FIELDS_ALLOWING_CSS.include?(field.to_s)
+        unfrozen_value = add_paragraphs_to_text(Sanitize.clean(fix_bad_characters(unfrozen_value),
+                               Sanitize::Config::CSS_ALLOWED.merge(transformers: transformers)))
```

Comment:
Notes for myself: `Sanitize::Config::CSS_ALLOWED` is to let class attributes exist in the first place, while `Sanitize::Transformers::ALLOW_USER_CLASSES` is to make sure those class attributes don't have weird values.